ui/src/pages/frontsite/components/ILSMenu/ILSMenu.js

```diff
@@ -1,22 +1,38 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 import { Container, Menu, Dropdown, Responsive } from 'semantic-ui-react';
-import {
-  authenticationService,
-  sessionManager,
-} from '@authentication/services';
+import { authenticationService } from '@authentication/services';
 import { FrontSiteRoutes, BackOfficeRoutes } from '@routes/urls';
-import { LoginButton } from '@components';
+import { LoginRedirectButton } from '@authentication/components';
+import { goTo } from '@history';
 
 export default class ILSMenu extends Component {
+  logout = async () => {
+    try {
+      const resp = await authenticationService.logout();
+      if (resp) {
+        this.props.fetchhUserProfile();
+        goTo(window.location.pathname);
+      }
+    } catch (e) {
+      this.props.sendErrorNotification(
+        'Logout',
+        'Something went wrong. Please try to logout again.'
+      );
+    }
+  };
+
   renderRightDropDown = userMenuText => {
     return (
       <Dropdown item text={userMenuText} icon="user">
         <Dropdown.Menu>
           <Dropdown.Item as={Link} to={FrontSiteRoutes.patronProfile}>
             Your Profile
           </Dropdown.Item>
-          {sessionManager.hasRoles(['admin', 'librarian']) ? (
+          {authenticationService.hasRoles(this.props.user, [
+            'admin',
+            'librarian',
```

Comment:
These roles should come from the `invenioConfig.js`

Reply:
We have hardcoded roles also in the `App.js` file for accessing the backoffice and thus I followed the same principle here? @ntarocco @kprzerwa @equadon do you think we need to move these roles in the config?

Reply:
Yeah, why not in the config, changes also `App.js`.